### ISSUE TITLE
It seems that the model was trained with only one subject?

### ISSUE THREAD

**Issue by nnzzll** [leohsuofnthu/Pytorch-IterativeFCN#7]
In `train.py`, you use `next(iter(train_loader))` to fetch the patches every iteration.

However , in such way you build an iterator of train_loader every iteration,and `next()` would return the first element of the iterator every time, which means during the whole iteration there is only one subject is used to training.

I'm just confused about this.Could anyone give me some explanation or discussion?

**Comment by nnzzll** [comment]
OK, I found out that when setting `shuffle=True` in DataLoader,  the `__getitem__` method will return random index which is different every first time.
